setup.cfg

```diff
@@ -3,12 +3,16 @@ exclude = .eggs,ENV,build,docs/conf.py,venv
 
 [yala]
 radon mi args = --min C
-pylint args = --disable=too-few-public-methods,too-many-instance-attributes,super-init-not-called
+pylint args = --disable=too-few-public-methods,too-many-instance-attributes,super-init-not-called,too-many-branches
 
 [pydocstyle]
-add-ignore = D105,D107
+add-ignore = D102,D105,D107,D104
 # D105: Missing docstring in magic method
 # D107: Missing docstring in __init__
```

Comment:
Please follow the same pattern and add a comment line describing what D104 stands for.

Reply:
Thanks. Solved

Reply:
Thanks. Solved.